homeassistant/components/homeassistant_alerts/__init__.py

```diff
@@ -170,6 +176,23 @@ async def _async_update_data(self) -> dict[str, IntegrationAlert]:
                     if self.ha_version >= resolved_in_version:
                         continue
 
+            if self.supervisor and "supervisor" in alert:
+                if (supervisor_info := get_supervisor_info(self.hass)) is None:
```

Comment:
When can get_supervisor_info be NOne?

Reply:
https://github.com/home-assistant/core/pull/82862#discussion_r1033576500